support-workers/src/main/scala/com/gu/emailservices/ContributionEmailFields.scala

```diff
@@ -27,6 +27,13 @@ case class ContributionEmailFields(
       "first payment date" -> formatDate(created.plusDays(10).toLocalDate),
       "payment method" -> "Direct Debit"
     )
+    case dd: ClonedDirectDebitPaymentMethod => List(
+      "account name" -> dd.bankTransferAccountName,
+      "sort code" -> hyphenate(dd.bankCode),
+      "Mandate ID" -> directDebitMandateId.getOrElse(""),
+      "first payment date" -> formatDate(created.plusDays(10).toLocalDate),
```

Comment:
Maybe extract this 10 to a named val? Or create a function which takes `created` and returns the first payment date